docs/configure/themes.md

```diff
@@ -0,0 +1,115 @@
+# :material-page-next-outline: Themes
+
+Knulli's default frontend is EmulationStation and default theme is [Art Book Next](https://github.com/anthonycaccese/art-book-next-es). Also included by default is the [Carbon](https://github.com/fabricecaruso/es-theme-carbon) theme from Batocera. Both are updated automatically as part of each release.
+
+## Preview
+
+<table>
+  <tr>
+    <td><img src="../../_inc/images/screenshots/system-view.png"/></td>
+    <td><img src="../../_inc/images/screenshots/menu.png"/></td>
+  </tr>
+  <tr>
+    <td><img src="../../_inc/images/screenshots/gamelist-view-metadata-immersive.png"/></td>
+    <td><img src="../../_inc/images/screenshots/gamelist-view-no-metadata-immersive.png"/></td>
+  </tr>
+</table>
+
+## Configuration
+
+The following options can be changed directly from the main menu under `User Interface Settings > Theme Configuration`
+
+| Setting | Description | Options |
+| -- | -- | -- |
+| Distribution | Used to define which folder to look in for Theme Customization files. | `Batocera/Knulli`, `RetroBat` |
+| Aspect Ratio | Enables you to select the correct aspect ratio for your screen.  This will automatically set itself so you should not need to change it but if the theme layout looks odd or spacing looks incorrect you can use this setting to make sure the aspect ratio matches your screen. | `16:9`, `16:10`, `4:3`, `3:2`, `1:1` |
+| System Artwork | Defines the set of artwork that is displayed on the system view | `Default`, `Noir`, `Custom`, `Custom (Fullscreen)` |
+| System Logos | Defines the logo set used on all views | `Default`, `Custom` |
+| Game Artwork | Defines the type of artwork used to represent a game. These are sourced from the the selections you make in the scraper menu. Image will display the image you selected to scrape for `Image Source`.  Image (Cropped) will display that same image zoomed in to fill the screen.  Boxart will display the image you selected to scrape for `Box Source` | `Image`, `Image (Cropped)`, `Boxart` |
```

Comment:
Maybe you could add a link to the scraping section here. Totally optional, it's just that I would've done it! Especially because the scraping section ties in with the theme, there's information about which media can be displayed by the theme. I think from here it should be `[scraper](../../play/scraping)`, but I'm not sure.